### ISSUE TITLE
Unable to pre-select options

### ISSUE THREAD

**Issue by onisiforos7** [abhimediratta/vue-multiselect-listbox#3]
Im filling **v-model** with the same structure array as in **options** array but i dont get the selected values in right column

**Comment by JuanAdamuz** [comment]
same here

**Comment by Yurifull** [comment]
Same request

**Comment by abhimediratta** [comment]
Can give an example of the `options` array and the `v-model` that you're passing?

**Comment by abhimediratta** [comment]
Whenever you use an array of objects, pass in both props: `reduceDisplayProperty` and `reduceValueProperty`.

Like in here:
```
<v-multiselect-listbox :options="[{label: 'Alabama', code: 'al'}, {label: 'California', code: 'cal'}]"
                       :reduce-display-property="(option) => option.label"
                       :reduce-value-property="(option) => option.code">
</v-multiselect-listbox>
```

Let me know if this resolves it.

**Comment by abhimediratta** [comment]
This has been fixed in the new version: 0.0.19. Thanks for reporting this. 
Apologies for the delay.

**Comment by Yurifull** [comment]
Thanks!!
I upgrade to v0.0.20 and works fine...

Congrats from Chile :)
